mlir/utils/performance/perfRunner.py

```diff
@@ -137,9 +150,40 @@ def getNanoSeconds(fileName):
         result = 0
         for row in reader:
             result += int(row['AverageNs'])
-        csv_file.close()
         return result
 
+def getNanoSecondsV2(fileName):
```

Comment:
Overall comment: perhaps `class Profiler:` is in order here, instead of hot-swapping functions onto variables?